public/scripts/controllers/AlbumsIndexController.js

```diff
@@ -0,0 +1,67 @@
+angular
+  .module('tunely')
+  .controller('AlbumsIndexController', AlbumsIndexController);
+
+AlbumsIndexController.$inject = ['$http'];
+
+function AlbumsIndexController ($http) {
+  var vm = this;
+  vm.newAlbum = {};
+  vm.newAlbum = {
+    name: 'Viva Hate',
+    artistName: 'Morrissey'
+  };
+
+  $http({
+    method: 'GET',
+    url: '/api/albums'
+  }).then(function successCallback(response) {
+    vm.albums = response.data;
+  }, function errorCallback(response) {
+    console.log('There was an error getting the data', response);
+  });
+
+  vm.createAlbum = function () {
+    $http({
+      method: 'POST',
+      url: '/api/albums',
+      data: vm.newAlbum,
+    }).then(function successCallback(response) {
+      vm.albums.push(response.data);
+    }, function errorCallback(response) {
+      console.log('There was an error posting the data', response);
+    });
+  }
+
+  vm.editAlbum = function (album) {
```

Comment:
IMO this controller is overloaded. Following the logic of separating out the `albumsShowController` I would argue that each individual CRUD action should also live in its own controllers. (But I understand that this would also require an Albums Service/Factory/Resource to tie it all together.)